### DISCUSSION TITLE
Integrate GVM 20.XX (GSE) into Sn1per

### DISCUSSION THREAD

**Discussion by fr0stedfl4ke** [1N3/Sn1per#338] [Ideas]
Based on all the recent changes with Greenbone and OpenVAS as a whole, I think it'd be a great idea to integrate GVM into Sn1per. Greenbone has completely rearchitected their scanner and overall workflow to the point where it requires less resources and is much more accurate out of the box. For those that don't have the capability of integrating Nessus Pro into Sn1per, GVM would be a great alternative. It seems like the kali repo's now support GVM too.

https://community.greenbone.net/t/about-gvm-20-08-architecture/8449
https://community.greenbone.net/t/gvm-20-08-stable-initial-release-2020-08-12/6312
https://community.greenbone.net/t/frequently-asked-questions-faq/5558

**Comment by 1N3** [comment]
This is something we've been looking into and are trying to implement. There is already a feature request here tracking it: https://github.com/1N3/Sn1per/issues/293

However, there have been several issues with the Kali native GVM packages and getting the integration to work properly. We had a working integration in Kali 2020.2, but due to some changes in Kali/GVM in 2021.1, the integration is no longer working. :( 

We'll continue to see what can be done for this, but given the complexity and time requirements, if we do get a working integration, it will likely be a paid add-on option for Sn1per Professional.

**Comment by 1N3** [comment]
GVM 21.x integration was finally added to Sn1per Professional v10.0. More info here: https://sn1persecurity.com/wordpress/sn1per-professional-v10-released/

**Comment by leogoldim** [comment]
Is there a plan to add this support in the Open Source version?
